keras_nlp/models/bert/bert_tasks.py

```diff
@@ -174,8 +291,11 @@ def from_preset(
                 f"""{", ".join(cls.presets)}. Received: {preset}."""
             )
 
+        if "preprocessor" not in kwargs:
+            kwargs["preprocessor"] = BertPreprocessor.from_preset(preset)
+
         # Check if preset is backbone-only model
-        if preset in cls.backbone_presets:
+        if preset in Bert.presets:
```

Comment:
This felt cleaner to me than the `backbone_presets` added in the last PR. The `Bert` class should be the thing that enumerates what can be called for `Bert.from_preset`.

Reply:
Sorry, already merged the PR! Perhaps one compromise could be 
```python
def backbone_presets(cls):
    return Bert.presets()
```

I think the property is needed otherwise it will be very hard to abstract out the `from_preset` logic.

Reply:
Talked a bit offline. We may want to factor out a common base class with `from_preset` logic. But even in that case, I think we could pull all this info from the backbone class.

I think we can remove this property for now (it makes it simpler to support the sst2 preset for tokenizer/preprocessor). And solve any issue related to common base as a follow up.